prog/detect/sensors-detect

```diff
@@ -2343,6 +2343,13 @@ use constant FEAT_SMBUS	=> (1 << 7);
 		devid_mask => 0xFFF0,
 		logdev => 0x0b,
 		features => FEAT_IN | FEAT_FAN | FEAT_TEMP,
+	}, {
+		name => "Nuvoton NCT6798D Super IO Sensors",
+		driver => "nct6775",
+		devid => 0xD428,
+		devid_mask => 0xFFF0,
```

Comment:
This doesn't look right. `SIO_ID_MASK` has been changed from 0xFFF0 to 0xFFF8 in the patch you linked. We'll have to do the same for `devid_mask` in sensors-detect for all the devices supported by the nct6775 driver. Otherwise sensors-detect would now detect both NCT6796 and NCT6798 on boards that have either of them.